server/resources_api/rank.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import os
+from sqlalchemy import create_engine, text, Table, MetaData, URL, insert, Column
+from sqlalchemy.dialects.mysql import VARCHAR
+from dotenv import load_dotenv
+
+
+# loads the variables in the .env file so we can access them
+load_dotenv()
+
+def get_database_uri(): 
+    uri = URL.create(
+        "mysql+mysqlconnector",
+        username=os.getenv("DB_USERNAME"),
+        password=os.getenv("DB_PASSWORD"),  # plain (unescaped) text
+        host=os.getenv("DB_SERVER"),
+        port= os.getenv("DB_PORT"),
+        database=os.getenv("DB_NAME"),
+    )
+    return uri
+
+# Create an SQLAlchemy engine
+engine = create_engine(get_database_uri())
+
+# Define the SQL create table statement
+sql_create = """CREATE TABLE IF NOT EXISTS uni_ranking_table (
+    uni_name VARCHAR(200) NOT NULL PRIMARY KEY,
+    uni_rank VARCHAR(10) NOT NULL,
+    uni_url VARCHAR(200)
+);"""
```

Comment:
This introduces a new step to the local setup, and it causes fragmentation to the code. I would prefer it to be the same way, but if not, find a way to put it in the `database_schemas/` directory.

You may put it in the a new `.sql` file or the same `schema.sql` file.